src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -89,57 +88,48 @@ void PurchaseButton_Click(object sender, EventArgs e)
 
         void getTrialLicenseButton_Click(object sender, EventArgs e)
         {
-            var parameterCollection = new NameValueCollection
-            {
-                {"NugetUser", IsNugetUser().ToString()},
-                {"PlatformInstaller", HasUserInstalledPlatform().ToString()},
-                {"TrialStartDate", TrialStartDateStore.GetTrialStartDate().ToString()}
-            };
 
-            UriBuilder builder;
+            string baseUrl;
             if (CurrentLicense != null && !CurrentLicense.IsExtendedTrial)
             {
                 // Original 14 day trial expired, give the user a chance to extend trial
-                builder = new UriBuilder("https://particular.net/extend-nservicebus-trial")
-                {
-                    Query = string.Join("&", parameterCollection.AllKeys.Select(key => $"{HttpUtility.UrlEncode(key)}={HttpUtility.UrlEncode(parameterCollection[key])}"))
-                };
+                baseUrl = "https://particular.net/extend-nservicebus-trial";
             }
             else
             {
                 // Extended trial license expired, ask the user to Contact Sales
-                builder = new UriBuilder("https://particular.net/extend-your-trial-45")
-                {
-                    Query = string.Join("&", parameterCollection.AllKeys.Select(key => $"{HttpUtility.UrlEncode(key)}={HttpUtility.UrlEncode(parameterCollection[key])}"))
-                };
+                baseUrl = "https://particular.net/extend-your-trial-45";
             }
 
+            var trialStart = HttpUtility.UrlEncode(TrialStartDateStore.GetTrialStartDate().ToString("yyyy-MM-dd", CultureInfo.InvariantCulture));
```

Comment:
@indualagarsamy   I think UrlEncode here is redundant.  The input would be the same as the output for that date string